docs/installation.md

```diff
@@ -2,422 +2,437 @@
-5. [Configuration](#configuration) shows you how to configure the *Backend*.
-6. [Startup](#startup) shows you how to start the *Backend*.
-7. [Post-installation sanity check](#post-installation-sanity-check) lets you verify that your *Backend* has started correctly.
+This document walks you through the installation of the Zonemaster *Backend* in
+the following sections:
+
+1. [Choices](#choices) lets you make all choices up front that this guide
+presumes you've made.  
+2. [Prerequisites](#prerequisites) lets you check all
+assumptions made by this guide about pre-installed software.  
+3. [Dependencies](#dependencies) shows you how to install all software that your
+Zonemaster *Backend* will depend on.  
+4. [Installation](#installation) shows you how to install the *Backend* itself.  
+5. [Configuration](#configuration) shows you the database and *Backend* configuration.  
+6. [Startup](#startup) shows you how to start the *Backend*.  
+7. [Post-installation sanitycheck](#post-installation-sanity-check) lets you 
+   verify that your *Backend* has started correctly.
 
 This document ends with appendices:
 
-* [Quick installation using a Docker container](#quick-installation-using-a-docker-container)
+* [Quick installation using a Docker container](#quick-installation-using-a-docker-container) 
 * [Administrative tasks](#administrative-tasks)
 
 
-## Choices
+## 1. Choices
 
-In various places throughout this guide you will be assumed to have made various choises.
-This section lists all those choices so you can make them up front.
+In various places throughout this guide you will be assumed to have made various
+choises.  This section lists all those choices so you can make them up front.
 
 
-### *Operating system*
+### 1.1 Operating system
 
-Zonemaster *Backend* needs to run on an operating system.
-This guide provides instructions for the following ones:
+Zonemaster *Backend* needs to run on an operating system. One can choose any of
+the following OS to install the *Backend*.
 
-* CentOS
-* Debian
-* FreeBSD
+* CentOS 
+* Debian 
+* FreeBSD 
 * Ubuntu
 
 
-### *Database engine*
+### 1.2 *Database engine*
 
-Zonemaster *Backend* needs a database engine.
-This guide provides instructions for the following ones:
+Zonemaster *Backend* needs a database engine.  The choice for the database are
+as follows :
 
-* MySQL
-* PostgreSQL (currently no instructions for CentOS)
+* MySQL 
+* PostgreSQL (currently no instructions for CentOS) 
 * SQLite (currently no instructions for any OS)
 
 
-## Prerequisites
+## 2. Prerequisites
 
-This guide assumes that the following softwares are already installed on the target system:
+This guide assumes that the following softwares are already installed on the
+target system:
 
-* the chosen operating system
-* sudo (only for installation and other administrative tasks)
-* cpanm (only for installation)
+* the chosen operating system sudo (only for installation and other
+  administrative tasks) 

```

Comment:
I think the original version is better:

* the chosen operating system
* sudo (only for installation and other administrative tasks)

You have to choose OS, and you have to install sudo unless the OS already has sudo (FreeBSD does not).